src/components/FluidCarousel/README.md

```diff
@@ -0,0 +1,299 @@
+	</div>
+</template>
+
+<script>
+import { MFluidCarousel, MFluidCarouselControls } from '@square/maker/components/FluidCarousel';
+
+const ITEM_SIZE = 250;
+const ITEM_MARGIN_SIZE = 10;
+const SCROLL_ITEMS = 2;
+
+export default {
+	components: {
+		MFluidCarousel,
+		MFluidCarouselControls,
+	},
+
+	computed: {
+
+		containerStyles() {
+			return {
+				'--carousel-item-size': `${ITEM_SIZE}px`,
+				'--carousel-item-margin-size': `${ITEM_MARGIN_SIZE}px`,
+			};
+		},
+
+		pixelsToScroll() {
+			return (ITEM_SIZE + ITEM_MARGIN_SIZE) * SCROLL_ITEMS;
+		},
+	},
+};
+</script>
+<style scoped>
+.container {
+	height: 700px;
+	width: var(--carousel-item-size);
+}
+
+.carousel-item {
+	width: var(--carousel-item-size);
+	height: var(--carousel-item-size);
+	background-color: rgba(0, 0, 0, 0.1);
+	margin-bottom: var(--carousel-item-margin-size);
+	user-select: none;
+	-webkit-user-select: none;
+  -ms-user-select: none;
+}
+.carousel-footer {
+	display: flex;
+	justify-content: center;
+	padding-top: 12px;
+}
+</style>
+
+```
+
+<!-- api-tables:start -->
+## FluidCarousel Props
+
+| Prop             | Type      | Default | Possible values | Description                                                                 |
+| ---------------- | --------- | ------- | --------------- | --------------------------------------------------------------------------- |
+| is-draggable     | `boolean` | `false` | —               | Whether the user can navigate the carousel by dragging                      |
+| item-count       | `number`  | `0`     | —               | Number of items in the carousel. Re-enables scroll:end event when increased |
+| is-vertical      | `boolean` | `false` | —               | Whether the carousel has a vertical layout - default is horizontal          |
+| pixels-to-scroll | `number`  | `0`     | —               | Number of pixels to scroll when using navigating using controls             |
+| snap-scroll      | `boolean` | `false` | —               | Whether scrolling snaps when navigating using touch controls                |
+
+
+## FluidCarousel Slots
+
+| Slot    | Description                       |
+| ------- | --------------------------------- |
+| header  | Slot for items above the carousel |
+| default | FluidCarousel items               |
+| footer  | Slot for items below the carousel |
+
+
+## FluidCarousel Events
+
+| Event                      | Type      | Description                                             |
+| -------------------------- | --------- | ------------------------------------------------------- |
+| overflowing                | `boolean` | Event emitted when carousel content overflows container |
+| carousel:mounted           | -         | Event emitted during mounted lifecycle hook             |
+| scroll:end                 | -         | Emitted when the scroll position is near the end        |
+| carousel:at-first-position | `boolean` | Emitted when carousel moves                             |
```

Comment:
as a general question / to open discussion, have we considered hiding the left/right controls when at the first/last position accordingly by default? it's a behavior we tend to add when using `FluidCarouselControls`, n i can't rly imagine a case in which we would be strongly opposed to that behavior.